admin-dev/themes/new-theme/js/pages/product/components/generator/AttributesSelector.vue

```diff
@@ -169,27 +196,30 @@
       },
     },
     methods: {
-      initDataSetConfig() {
+      initDataSetConfig(): void {
         const searchItems = this.getSearchableAttributes();
         this.searchSource = new Bloodhound({
-          datumTokenizer: Tokenizers.obj.letters(
+          datumTokenizer: Tokenizers.obj.letters([
             'name',
             'value',
             'group_name',
+          ] as any,
```

Comment:
Why changing 3 parameters in an array ?

Reply:
I fixed it but having 3 value is an error anyway, `Tokenizers.obj.letters` takes only 1 value